woocommerce/woocommerce-android
# Issue/9706 onboarding wcpay instruction

Closes: https://github.com/woocommerce/woocommerce-android/issues/9706


This PR builds the two screen below:

<img width="569" alt="Screenshot 2023-08-30 at 20 39 18" src="https://github.com/woocommerce/woocommerce-android/assets/266376/cb80cd12-6ff7-414f-be83-bf3b0da810ef">

### Description


This PR adds the two screens above. Each screen is just a combo of fragment and Compose, with no viewmodels, since there's no states to handle. All click handlings are added in the fragments.

### Testing instructions

1. Start `woocommerce-payments`
2. Check the 2 steps shown before navigating to the webview flow match the Figma design from the screenshots
3. Click on all the links from the 2 steps and check that the correct url is loaded.

### Images/gif



https://github.com/woocommerce/woocommerce-android/assets/2663464/494ad216-5b5c-4a19-8a7c-81111db006fd
